### PR TITLE
Fix storage min references

### PR DESCRIPTION
**Pull request by prosif**
Show min_cpu and memory_min in image_version serializer, change references and checks from storage_min to cpu_min.

### CODE REVIEW

File: core/migrations/0033_auto_20151104_1452.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0032_auto_20150925_1239'),
```
Comment (steve-gregory) [inline core/migrations/0033_auto_20151104_1452.py]: Rename these auto migrations to something meaningful, please :+1:

Comment (lenards) [inline core/migrations/0033_auto_20151104_1452.py]: Yes! :+1:

File: core/migrations/0038_threshold_rename_storagemin_to_cpumin.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('core', '0032_auto_20150925_1239'),
+    ]
```
Comment (lenards) [inline core/migrations/0038_threshold_rename_storagemin_to_cpumin.py]: @prosif - you're going to need to edit this `dependencies` for your migration at well. 

So fall in line with the rest of the migration, I believe this should be: 
```python
    dependencies = [
        ('core', '0037_add_end_date_identitymembership'),
    ]
```

Comment (lenards) [inline core/migrations/0038_threshold_rename_storagemin_to_cpumin.py]: @steve-gregory - can you sanity check what I said above :arrow_up:

Comment (steve-gregory) [inline core/migrations/0038_threshold_rename_storagemin_to_cpumin.py]: :+1: Dependency must change (0032_auto doesn't exist, anyway)

Comment (lenards) [inline core/migrations/0038_threshold_rename_storagemin_to_cpumin.py]: Meaningful migration names :white_check_mark:

**Comment by steve-gregory** [comment]
:shipit:

**Pull request merged**